jsonnet/kube-prometheus/blackbox-exporter/blackbox-exporter.libsonnet

```diff
@@ -0,0 +1,211 @@
+                  ports: [{
+                    name: 'http',
+                    containerPort: bb.port,
+                  }],
+                  resources: {
+                    requests: $._config.resources['blackbox-exporter'].requests,
+                    limits: $._config.resources['blackbox-exporter'].limits
+                  },
+                  securityContext: if bb.privileged then {
+                                     runAsNonRoot: false,
+                                     capabilities: { drop: [ 'ALL' ], add: [ 'NET_RAW'] }
+                                   } else {
+                                     runAsNonRoot: true,
+                                     runAsUser: 65534
+                                   },
+                  volumeMounts: [{
+                    mountPath: '/etc/blackbox_exporter/',
+                    name: 'config',
+                    readOnly: true
+                  }]
+                },
+                {
+                  name: 'module-configmap-reloader',
+                  image: $._config.imageRepos.configmapReloader + ':' + $._config.versions.configmapReloader,
+                  args: [
+                    '--webhook-url=http://localhost:' + bb.port + '/-/reload',
+                    '--volume-dir=/etc/blackbox_exporter/'
+                  ],
+                  resources: {
+                    requests: $._config.resources['blackbox-exporter'].requests,
+                    limits: $._config.resources['blackbox-exporter'].limits
+                  },
+                  securityContext: { runAsNonRoot: true, runAsUser: 65534 },
+                  terminationMessagePath: '/dev/termination-log',
+                  terminationMessagePolicy: 'FallbackToLogsOnError',
+                  volumeMounts: [{
+                    mountPath: '/etc/blackbox_exporter/',
+                    name: 'config',
+                    readOnly: true
+                  }]
+                }
+              ],
+              nodeSelector: { 'kubernetes.io/os': 'linux' },
+              serviceAccountName: 'blackbox-exporter',
+              volumes: [{
+                name: 'config',
+                configMap: { name: 'blackbox-exporter-configuration' }
+              }]
+            }
+          }
+        }
+      },
+
+      service: {
+        apiVersion: 'v1',
+        kind: 'Service',
+        metadata: {
+          name: 'blackbox-exporter',
+          namespace: $._config.namespace,
+          labels: bb.assignLabels,
+        },
+        spec: {
+          ports: [{ name: 'http', port: bb.port, targetPort: 'http' }],
+          selector: bb.matchLabels,
+        }
+      },
+
+      serviceMonitor:
+        {
+          apiVersion: 'monitoring.coreos.com/v1',
+          kind: 'ServiceMonitor',
+          metadata: {
+            name: 'blackbox-exporter',
+            labels: bb.assignLabels
```

Comment:
Thx, for your work. Just a small question here. Is there a reason, why the namespace field is missing here?

Reply:
Because I wasn't paying enough attention, apparently. I must've missed it when I got rid of `ksonnet` (before posting the PR). Good catch, thank you.